android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java

```diff
@@ -51,8 +56,12 @@
 	private float shadowX = 0f;
 	private float shadowY = 0f;
 	private int shadowColor = Color.TRANSPARENT;
-	private String minimumFontSize = null;
-	private String autoshrinkSetFontSize = null;
+	private int autoLinkFlags;
+	private int viewHeightInLines;
+	private int maxLines = Integer.MAX_VALUE;
+	private float minimumFontSizeInPixels = -1.0f;
```

Comment:
There are 2 reasons why I made it this way.

1) Font sizes in JavaScript can be set in units other than pixels by the developer. They can be set in dips, inches, millimeters, and centimeters. By default, font units are in dips in Titanium. So, odds are, all font sizes will be stored in fractional form when converted to pixels.

2) Android supports fractional font sizes... even in pixels. So do Apple and Microsoft. It's actually noticeable on low DPI screens because it affects how the edges of the text are alpha-blended to the nearest whole screen/bitmap pixel.